server.go

```diff
@@ -242,6 +242,10 @@ func KeepaliveEnforcementPolicy(kep keepalive.EnforcementPolicy) ServerOption {
 // CustomCodec returns a ServerOption that sets a codec for message marshaling and unmarshaling.
 //
 // This will override any lookups by content-subtype for Codecs registered with RegisterCodec.
+//
+// Deprecated: server side codec can be registered, and it will be picked by the
```

Comment:
`Deprecated: register codecs using encoding.RegisterCodec.  The server will automatically use registered codecs based on the incoming requests' headers` ?

Reply:
...`see also <link>`

Reply:
Done